mergify_engine/merge_train.py

```diff
@@ -198,7 +198,13 @@ async def _delete_branch(self) -> None:
 
     async def _report_failure(self, exception: http.HTTPClientSideError) -> None:
         title = "This pull request cannot be embarked for merge"
-        summary = exception.message
+
+        if self.queue_pull_request_number is None:
+            summary = "The merge-queue embarked pull request can't be created"
+        else:
+            summary = f"The merge-queue embarked pull request (#{self.queue_pull_request_number}) can't be prepared"
```

Comment:
```suggestion
            summary = "The merge-queue pull request can't be created"
        else:
            summary = f"The merge-queue pull request (#{self.queue_pull_request_number}) can't be prepared"
```

I already removed the `embarked` word in the first review, it does not really sound English otherwise.